src/app/domain/geolocation/tracking/storage.js

```diff
@@ -0,0 +1,143 @@
+}
+
+export const clearAllData = async () => {
+  await clearAllCozyGPSMemoryData()
+  await AsyncStorage.remove(StorageKeys.ShouldBeTrackingFlagStorageAdress)
+}
+
+export const getShouldStartTracking = async () => {
+  return await getData(StorageKeys.ShouldBeTrackingFlagStorageAdress)
+}
+
+export const clearAllActivities = async () => {
+  return AsyncStorage.removeItem(StorageKeys.Activities)
+}
+
+export const getLastPointUploaded = async () => {
+  return await getData(StorageKeys.LastPointUploadedAdress)
+}
+
+export const setLastPointUploaded = async value => {
+  await storeData(StorageKeys.LastPointUploadedAdress, value)
+}
+
+export const setLastStopTransitionTs = async timestamp => {
+  await storeData(StorageKeys.LastStopTransitionTsKey, timestamp.toString())
+}
+
+export const setLastStartTransitionTs = async timestamp => {
+  await storeData(StorageKeys.LastStartTransitionTsKey, timestamp.toString())
+}
+
+export const getLastStopTransitionTs = async () => {
+  const ts = await getData(StorageKeys.LastStopTransitionTsKey)
+  return ts ? parseInt(ts, 10) : 0
+}
+
+export const getLastStartTransitionTs = async () => {
+  const ts = await getData(StorageKeys.LastStartTransitionTsKey)
+  return ts ? parseInt(ts, 10) : 0
+}
+
+export const storeFlagFailUpload = async Flag => {
+  try {
+    await storeData(
+      StorageKeys.FlagFailUploadStorageAdress,
+      Flag ? 'true' : 'false'
+    )
+  } catch (error) {
+    Log('Error while storing FlagFailUpload:' + error.toString())
+    throw error
+  }
+}
+
+export const getFlagFailUpload = async () => {
+  try {
+    let value = await getData(StorageKeys.FlagFailUploadStorageAdress)
+    if (value == undefined) {
+      await storeFlagFailUpload(false)
+      return false
+    } else {
+      return value == 'true'
+    }
+  } catch (error) {
+    Log('Error while getting FlagFailUpload:' + error.toString())
+    throw error
+  }
+}
+
+export const getId = async () => {
+  return await getData(StorageKeys.IdStorageAdress)
+}
+
+export const storeId = async Id => {
+  await storeData(StorageKeys.IdStorageAdress, Id)
+}
+
+const setActivities = async activities => {
+  return storeData(StorageKeys.Activities, activities)
+}
+
+export const storeActivity = async activity => {
+  Log('Lets store activity : ' + JSON.stringify(activity))
+  if (!activity) {
+    return null
+  }
+  const activities = await getActivities()
+  if (activities?.length > 0) {
+    activities.push(activity)
+    await setActivities(activities)
+  } else {
+    await setActivities([activity])
+  }
+}
+
+export const getActivities = async ({ beforeTs } = {}) => {
+  const activities = await getData(StorageKeys.Activities)
+  if (beforeTs) {
+    const activitiesBeforeTs = activities.filter(activity => {
+      return activity?.data?.ts <= beforeTs
+    })
+    return activitiesBeforeTs
+  }
+  return activities
+}
+
+export const removeActivities = async ({ beforeTs } = {}) => {
+  console.log('remove activities')
```

Comment:
If you want to keep it, use the `Log` which uses cozy-minilog logger

Reply:
Yep sorry, the rebase was not perfect